test/cppsuite/tests/test_template.cpp

```diff
@@ -28,60 +28,84 @@
+        workload_tracking::populate_tracking_cursor(
+          operation, collection_id, key, value, ts, op_track_cursor);
+    }
+};
+
 /*
  * Class that defines operations that do nothing as an example. This shows how database operations
  * can be overridden and customized.
  */
 class test_template : public test_harness::test {
     public:
-    test_template(const test_harness::test_args &args) : test(args) {}
+    test_template(const test_harness::test_args &args) : test(args)
+    {
+        delete this->_workload_tracking;
+        this->_workload_tracking =
+          new tracking_table_template(_config->get_subconfig(WORKLOAD_TRACKING),
+            _config->get_bool(COMPRESSION_ENABLED), *_timestamp_manager);
+    }
 
     void
     run() override final
     {
-        /* You can remove the call to the base class to fully customized your test. */
+        /* You can remove the call to the base class to fully customize your test. */
         test::run();
     }
 
     void
-    populate(test_harness::database &, test_harness::timestamp_manager *,
-      test_harness::configuration *, test_harness::workload_tracking *) override final
+    populate(test_harness::database &database, test_harness::timestamp_manager *tsm,
+      test_harness::configuration *config, test_harness::workload_tracking *tracking) override final
     {
         std::cout << "populate: nothing done." << std::endl;
     }
 
     void
-    custom_operation(test_harness::thread_context *) override final
+    custom_operation(test_harness::thread_context *tc) override final
     {
         std::cout << "custom_operation: nothing done." << std::endl;
     }
 
     void
-    insert_operation(test_harness::thread_context *) override final
+    insert_operation(test_harness::thread_context *tc) override final
     {
         std::cout << "insert_operation: nothing done." << std::endl;
     }
 
     void
-    read_operation(test_harness::thread_context *) override final
+    read_operation(test_harness::thread_context *tc) override final
     {
         std::cout << "read_operation: nothing done." << std::endl;
     }
 
     void
-    remove_operation(test_harness::thread_context *) override final
+    remove_operation(test_harness::thread_context *tc) override final
     {
         std::cout << "remove_operation: nothing done." << std::endl;
     }
 
     void
-    update_operation(test_harness::thread_context *) override final
+    update_operation(test_harness::thread_context *tc) override final
     {
         std::cout << "update_operation: nothing done." << std::endl;
     }
 
     void
-    validate(const std::string &, const std::string &, const std::vector<uint64_t> &) override final
+    validate(bool custom_tracking_table, const std::string &operation_table_name,
+      const std::string &schema_table_name,
+      const std::vector<uint64_t> &known_collection_ids) override final
     {
         std::cout << "validate: nothing done." << std::endl;
```

Comment:
Should we assert that `custom_tracking_table` is `true` here?

Reply:
I'm not sure it adds much benefit here as users are expected to overwrite the function, and technically users can write custom validation but still use the default tracking table schema. 

I think leave as it is but can add if you wanna push for it.